integration/integration_api_test.go

```diff
@@ -35,6 +35,9 @@ func TestPiperGithubPublishRelease(t *testing.T) {
 	defer os.RemoveAll(dir) // clean up
 	assert.NoError(t, err, "Error when creating temp dir")
 
+	testAsset := filepath.Join(dir, "test.txt")
+	err = ioutil.WriteFile(testAsset, []byte("Test"), 0644)
```

Comment:
err should be checked or ignored explicitly.

Reply:
is checked now ...